test/python/basic.py

```diff
@@ -12,7 +12,9 @@
 
 # not ideal, but this solution is simple and portable. Alternative is to set
 # PYTHONPATH or test on installed sources.
-sys.path.insert(1, '../include')
+from pathlib import Path
+ur_module_path = str(Path(os.path.realpath(__file__)).parents[2].joinpath("include"))
```

Comment:
This is too hacky :-) Maybe instead we can just remove this entirely and set PYTHONPATH like this in the CMake:
```
set_property(TEST python-basic APPEND PROPERTY
                ENVIRONMENT "PYTHONPATH=${PROJECT_SOURCE_DIR}")
```
?

this should allow us to just `import include.ur as ur` instead.